server/auth/github/client.go

```diff
@@ -173,6 +177,29 @@ func (c *Client) IsMember(organization, token string) (bool, error) {
 	return true, nil
 }
 
+// IsTeamMember returns true if the given user is a member of the team.
+func (c *Client) IsTeamMember(teamId, name string, token string) (bool, error) {
```

Comment:
Kinda feels like the method signature should be `(teamID, token string)` and then just fetch the github user to grab their login first. It's an extra call, but `empire.User.Name` may not always be their GitHub login in the future.